### ISSUE TITLE
CNN model accuracy saturated 

### ISSUE THREAD

**Issue by suhail017** [suhail017/KCC_Work#3]
The model accuracy gets up to 80% and gets saturated afterward. Different pre-trained models with hyper-parameter tuning methods are currently applied to the models but still have the same. Might try some weight variations technique.
